thali/thalicryptomanager.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var crypto = require('crypto');
+var fs = require('fs');
+var path = require('path');
+
+var pkcs12FileName = '/pkcs12.pfx';
+var password = 'password';
+var certname = 'certname';
+var country = 'country';
+var organization = 'organization';
+var macName = 'SHA256';
+var hashSizeInBytes = 16;
+
+module.exports = {
+
+/**
+* Creates a PKCS12 content, saves it to a file, extracts the public
+* key and returns it's SHA256 hash value.
```

Comment:
Why doesn't the JSDoc specify the callback? See http://usejsdoc.org/tags-callback.html